metadata.rb

```diff
@@ -3,7 +3,7 @@
 maintainer_email  'help@sous-chefs.org'
 license           'Apache-2.0'
 description       'Installs and configures haproxy'
-version           '8.1.1'
+version           '8.1.2'
```

Comment:
Please revert version to 8.1.1
```suggestion
version           '8.1.1'
```

Reply:
Done